lib/auth/oidc_test.go

```diff
@@ -86,48 +86,70 @@ func (s *OIDCSuite) TestCreateOIDCUser(c *check.C) {
 		roles:         []string{"admin"},
 		sessionTTL:    1 * time.Minute,
 	})
-	c.Assert(err, check.IsNil)
+	require.NoError(t, err)
 
 	// Within that 1 minute period the user should still exist.
 	_, err = s.a.GetUser("foo@example.com", false)
-	c.Assert(err, check.IsNil)
+	require.NoError(t, err)
 
 	// Advance time 2 minutes, the user should be gone.
 	s.c.Advance(2 * time.Minute)
 	_, err = s.a.GetUser("foo@example.com", false)
-	c.Assert(err, check.NotNil)
+	require.Error(t, err)
 }
 
-// TestUserInfo ensures that an insecure userinfo endpoint returns
+// TestUserInfoBlockHTTP ensures that an insecure userinfo endpoint returns
 // trace.NotFound similar to an invalid userinfo endpoint. For these users,
 // all claim information is already within the token and additional claim
 // information does not need to be fetched.
-func (s *OIDCSuite) TestUserInfo(c *check.C) {
+func TestUserInfoBlockHTTP(t *testing.T) {
+	s := setUpSuite(t)
 	// Create configurable IdP to use in tests.
-	idp := newFakeIDP()
-	defer idp.Close()
+	idp := newFakeIDP(t, false)
 
 	// Create OIDC connector and client.
 	connector, err := types.NewOIDCConnector("test-connector", types.OIDCConnectorSpecV3{
 		IssuerURL:    idp.s.URL,
 		ClientID:     "00000000000000000000000000000000",
 		ClientSecret: "0000000000000000000000000000000000000000000000000000000000000000",
 	})
-	c.Assert(err, check.IsNil)
-	oidcClient, err := s.a.getOrCreateOIDCClient(connector)
-	c.Assert(err, check.IsNil)
+	require.NoError(t, err)
+	oidcClient, err := s.a.getOrCreateOIDCClient(connector, false)
```

Comment:
```suggestion
	oidcClient, err := s.a.getOrCreateOIDCClient(connector, false /* insecure */)
```

So on for similar occurrences.